make test fails on macOS

**Describe the bug**
A clear and concise description of what the bug is.
make test fails on macOS. See the details failure log in steps to reproduce section
**To Reproduce**
Steps to reproduce the behavior:
```
$ make test
go test -mod=readonly ./...
?       github.com/redhat-developer/kam/cmd/kam [no test files]
ok      github.com/redhat-developer/kam/pkg/cmd 5.787s
ok      github.com/redhat-developer/kam/pkg/cmd/environment     3.320s
?       github.com/redhat-developer/kam/pkg/cmd/genericclioptions       [no test files]
ok      github.com/redhat-developer/kam/pkg/cmd/service 4.843s
ok      github.com/redhat-developer/kam/pkg/cmd/ui      (cached)
ok      github.com/redhat-developer/kam/pkg/cmd/utility (cached)
?       github.com/redhat-developer/kam/pkg/cmd/version [no test files]
ok      github.com/redhat-developer/kam/pkg/cmd/webhook 7.523s
ok      github.com/redhat-developer/kam/pkg/pipelines   15.291s
ok      github.com/redhat-developer/kam/pkg/pipelines/argocd    2.650s
?       github.com/redhat-developer/kam/pkg/pipelines/argocd/operator/v1alpha1  [no test files]
?       github.com/redhat-developer/kam/pkg/pipelines/argocd/v1alpha1   [no test files]
?       github.com/redhat-developer/kam/pkg/pipelines/clientconfig      [no test files]
ok      github.com/redhat-developer/kam/pkg/pipelines/config    2.148s
ok      github.com/redhat-developer/kam/pkg/pipelines/deployment        (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/dryrun    (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/environments      6.482s
ok      github.com/redhat-developer/kam/pkg/pipelines/eventlisteners    2.416s
ok      github.com/redhat-developer/kam/pkg/pipelines/git       4.508s
?       github.com/redhat-developer/kam/pkg/pipelines/helper    [no test files]
ok      github.com/redhat-developer/kam/pkg/pipelines/imagerepo 2.668s
?       github.com/redhat-developer/kam/pkg/pipelines/ioutils   [no test files]
ok      github.com/redhat-developer/kam/pkg/pipelines/meta      (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/namespaces        (cached)
?       github.com/redhat-developer/kam/pkg/pipelines/pipelines [no test files]
ok      github.com/redhat-developer/kam/pkg/pipelines/resources (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/roles     (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/routes    (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/scm       1.221s
ok      github.com/redhat-developer/kam/pkg/pipelines/secrets   (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/statustracker     (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/tasks     (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/triggers  (cached)
ok      github.com/redhat-developer/kam/pkg/pipelines/webhook   0.901s
--- FAIL: TestWriteResources (0.03s)
    --- FAIL: TestWriteResources/Path_without_permission (0.00s)
        resources_test.go:45: error mismatch: got failed to MkDirAll for /test/myfile.yaml: mkdir /test: read-only file system, want failed to MkDirAll for /test/myfile.yaml: mkdir /test: permission denied
FAIL
FAIL    github.com/redhat-developer/kam/pkg/pipelines/yaml      1.008s
?       github.com/redhat-developer/kam/tools/cmd-docs  [no test files]
FAIL
make: *** [test] Error 1
```
**Expected behavior**
A clear and concise description of what you expected to happen.
make test should not report any error on macOS
**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/pkg/pipelines/yaml/resources_test.go
+++ b/pkg/pipelines/yaml/resources_test.go
@@ -35,7 +35,7 @@ func TestWriteResources(t *testing.T) {
 	}{
 		{"Path with ~", "~/manifest", ""},
 		{"Path without ~", filepath.Join(path, "manifest/gitops"), ""},
-		{"Path without permission", "/", "failed to MkDirAll for /test/myfile.yaml: mkdir /test: permission denied"},
+		{"Path without permission", "/", "failed to MkDirAll for /test/myfile.yaml"},
 	}
 
 	for _, test := range tests {

